### DISCUSSION TITLE
Google AI 有多厲害？

### DISCUSSION THREAD

**Discussion by GuanYann** [1101-GS4719-C/Python-Community-GS4719-C#1444] [General]
如今AI發展的非常迅速，醫療檢定考試能和專業醫生不相上下。AI強大的能力的確不可否認，即便如此，我認為醫生這個職業依舊不會被AI取代，AI是透過吸收大量的數據，並分析而得到的結果，而病人卻可能會有不同的個案，是AI無法掌握的，這時就必須靠醫生的臨場反應、應變，給予不同的病人不同的治療。但一些小感冒、小症狀，我覺得相比嚴重的疾病，AI提供的建議、診斷，還是多多少少可以拿來參考。

https://3c.ltn.com.tw/news/52482

**Comment by ThanatosJun** [comment]
醫生都跟AI合作，這是1+1大於2的概念。AI補足醫生的盲區。

**Comment by sjh501** [comment]
我之前有看到一個影片是在說醫生的用途不僅是在看病，而是有透過和病人的互動聊天，讓病人有被關心的感覺，這種心理上面的治療和關心我覺得還是很難被取代，但確實AI有其優點

**Comment by Tzuhsuannnnn** [comment]
AI在判斷上也許可以比醫生更精準更仔細，但在陪伴及關心上，我認為還是需要醫生的存在的。

**Comment by youtinglin** [comment]
雖然AI的發展，讓醫療技術更為精進，但更多的是作為醫生們的輔助工具，有時的診治判斷仍要除了專業角度判斷外，還要有人性化的考量，或者去因應病人的需求去治療。

**Comment by juliaouo** [comment]
AI 可能可以提供初步的診斷結果作為參考，但有些非典型案例 AI 可能無法精準判斷，這時候就需要醫生來判斷，降低誤診的機率，所以我認為醫生還是非常重要的存在，另外 AI 提供的診斷也需要人來負責，給病人交代。

**Comment by ashleyy-huang** [comment]
我認為AI對於醫學方面只可以是作為輔助的工具，沒辦法完全取代掉醫生。凡事總會有例外，若某一天突然出現超級病毒，也許機器人會沒辦法針對這樣新的東西做出醫療服務，這時候我們還是需要有真正的醫生來進行看診

**Comment by bestbanana295** [comment]
雖然運用AI可以做到很多事，但是有許多現實上的的困難是需要真人去處理的，就上樓所說的如果哪天出現了意外，就需要人類來主導。
